randovania/games/dread/json_data/header.json

```diff
@@ -942,87 +942,87 @@
                 "extra": {}
             },
             "Combat": {
                 "long_name": "Combat",
-                "description": "",
+                "description": "If this trick is enabled, the player may be expected to defeat enemies and bosses with fewer items and less health.",
                 "extra": {}
             },
             "Pseudo": {
                 "long_name": "Pseudo-Wave Beam",
-                "description": "",
+                "description": "It\u00b4s possible to fire through solid walls without obtaining wave beam.",
                 "extra": {}
             },
             "IBJ": {
                 "long_name": "Infinite Bomb Jump",
-                "description": "",
+                "description": "By chaining and timing bomb jumps it`s possible to reach the top of a room.",
                 "extra": {}
             },
             "HBJ": {
                 "long_name": "Horizontal Bomb Jump",
-                "description": "",
+                "description": "Standing on a bomb while morphed will blast you in a certain direction. This also works midair.",
                 "extra": {}
             },
             "WBJ": {
                 "long_name": "Water Bomb Jump",
-                "description": "",
+                "description": "Performing a WBJ will make higher bomb jumps underwater possible. To setup a WBJ you need to bomb jump under water and demorph midair. If you now perform a bomb jump midair without touching the ground as morphball you\u00b4ll be blasted higher than normal.",
                 "extra": {}
             },
             "WSJ": {
                 "long_name": "Water Space Jump",
-                "description": "",
+                "description": "Used to gain height underwater in certain places without Gravity Suit.",
                 "extra": {}
             },
             "SWJ": {
                 "long_name": "Single-wall Wall Jump",
-                "description": "",
+                "description": "With this technique its possible to jump up a single wall all the way up.",
                 "extra": {}
             },
             "Slide": {
                 "long_name": "Slide Jump",
-                "description": "",
+                "description": "By sliding off a cliff and jumping right before you fall you\u00b4ll jump further.",
                 "extra": {}
             },
             "Speedbooster": {
                 "long_name": "Speedbooster Conservation",
-                "description": "",
+                "description": "Activating the speedbooster will allow you to destroy speedblocks.",
                 "extra": {}
             },
             "Walljump": {
                 "long_name": "Walljump",
-                "description": "",
+                "description": "Basic movement ability which can be abused in not intended ways.",
```

Comment:
Still should technically say "unintended" rather than "not intended". you marked it as resolved without actually resolving it.